src/python/doc/alpha_complex_user.rst

```diff
@@ -177,26 +173,86 @@ of speed-up, since we always first build the full filtered complex, so it is rec
 :paramref:`~gudhi.AlphaComplex.create_simplex_tree.max_alpha_square`.
 In the following example, a threshold of :math:`\alpha^2 = 32.0` is used.
 
+Weighted version
+^^^^^^^^^^^^^^^^
+
+A weighted version for Alpha complex is available. It is like a usual Alpha complex, but based on a
+`CGAL regular triangulation <https://doc.cgal.org/latest/Triangulation/index.html#title20>`_.
+
+This example builds the CGAL weighted alpha shapes from a small molecule, and initializes the alpha complex with
```

Comment:
This sentence is confusing. Maybe just "This example builds the weighted alpha-complex of a small molecule, where atoms have different sizes. It is taken from ..."?

Reply:
I reworded it on 1f9b9d2e